src/common/config_opts.h

```diff
@@ -1680,6 +1680,4 @@ OPTION(internal_safe_to_start_threads, OPT_BOOL, false)
 
 OPTION(debug_deliberately_leak_memory, OPT_BOOL, false)
 
-//This option is to return size of all segments for the manifest in DLO during bucket listing
-OPTION(rgw_swift_dlo_all_segments_size, OPT_BOOL, false)
-OPTION(rgw_swift_custom_header, OPT_STR, "")
+OPTION(rgw_swift_custom_header, OPT_STR, "") // option to enable swift custom headers
```

Comment:
@prallabh squash this change into the first commit, rewrite first commit title and message